view/theme/quattro/style.php

```diff
@@ -5,27 +5,30 @@
 
 	$uid = get_theme_uid();
 
-	$color=false;
-	$quattro_align=false;
-	$site_color = Config::get("quattro","color");
-	$site_quattro_align = Config::get("quattro", "align" );
-	
+	$color = false;
+	$quattro_align = false;
+	$site_color = Config::get("quattro", "color", "dark);
+	$site_quattro_align = Config::get("quattro", "align", false);
+
 	if ($uid) {
-		$color = PConfig::get( $uid, "quattro","color");
-		$quattro_align = PConfig::get( $uid, 'quattro', 'align' );
-	}
-	
-	if ($color===false) $color=$site_color;
-	if ($color===false) $color="dark";
-	if ($quattro_align===false) $quattro_align=$site_quattro_align;
-	
-		
+		$color = PConfig::get($uid, "quattro", "color");
+		$quattro_align = PConfig::get($uid, 'quattro', 'align');
+	}
+
+	if ($color === false) {
```

Comment:
Isn't this supposed to be `null` if the `PConfig::get` above call can't find the quattro/color config value?